### PR TITLE
Updating the Automatic Run Triggering and VCS Branch settings docs to…

### PR DESCRIPTION
**Pull request by salilsub**
… make it clear that runs will trigger on all Git tags, regardless of VCS Branch setting

### What
Developer/Terraform/Terraform Cloud/Workspaces/Settings/VCS Connections

### Why
The documentation is not explicit that if the git tags setting is enabled that the VCS Branch setting will be ignored. Both the Git Tags and VCS Branch sections of the settings docs were updated to reflect this.

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/workspaces/settings/vcs.mdx [hunk 1]
```diff
@@ -83,9 +83,11 @@ You must include an additional `\` to escape the regex pattern when you manage y
 | **Version contains a prefix** | `\d+.\d+.\d+$`            | `\\d+.\\d+.\\d+$`        |
 | **Version contains a suffix** | `^\d+.\d+.\d+`            | `^\\d+.\\d+.\\d+`        |
 
+Runs will be triggered on all tags matching this pattern, regardless of the value in the [VCS Branch](#vcs-branch) setting
```
Comment (laurapacilio) [inline website/docs/cloud-docs/workspaces/settings/vcs.mdx:86]:
```suggestion
Terraform Cloud triggers runs for all tags matching this pattern, regardless of the value in the [VCS Branch](#vcs-branch) setting.
```

File: website/docs/cloud-docs/workspaces/settings/vcs.mdx [hunk 2]
```diff
@@ -83,9 +83,11 @@ You must include an additional `\` to escape the regex pattern when you manage y
...
+
 ### VCS Branch
 
-This designates which branch of the repository Terraform Cloud should use in the workspace. If left blank, Terraform Cloud uses the repository's default branch.
+This designates which branch of the repository Terraform Cloud should use in the workspace if you are using [Always Trigger Runs](#always-trigger-runs) or [Only trigger runs when files in specified paths change](#only-trigger-runs-when-files-in-specified-paths-change). If left blank, Terraform Cloud uses the repository's default branch. If you are triggering runs when a [git tag is published](#trigger-runs-when-a-git-tag-is-published), all tags will trigger runs, regardless of what this field is set to.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/workspaces/settings/vcs.mdx:90]:
```suggestion
This setting designates which branch of the repository Terraform Cloud should use when the workspace is set to [Always Trigger Runs](#always-trigger-runs) or [Only trigger runs when files in specified paths change](#only-trigger-runs-when-files-in-specified-paths-change). If you leave this setting blank, Terraform Cloud uses the repository's default branch. If the workspace is set to trigger runs when a [git tag is published](#trigger-runs-when-a-git-tag-is-published), all tags will trigger runs, regardless of the branch specified in this setting.
```

**Review by laurapacilio — APPROVED** [review]
Approved with a couple of suggested edits - thank you! :)  The edits are to make sure we're using active voice (Terraform Cloud triggers runs) and to adhere to our style guide (referring to functionality in the present tense - no "will".

File: website/docs/cloud-docs/workspaces/settings/vcs.mdx [hunk 3]
```diff
@@ -83,9 +83,11 @@ You must include an additional `\` to escape the regex pattern when you manage y
 | **Version contains a prefix** | `\d+.\d+.\d+$`            | `\\d+.\\d+.\\d+$`        |
 | **Version contains a suffix** | `^\d+.\d+.\d+`            | `^\\d+.\\d+.\\d+`        |
 
+HCP Terraform triggers runs for all tags matching this pattern, regardless of the value in the [VCS Branch](#vcs-branch) setting.
+
 ### VCS Branch
 
-This designates which branch of the repository Terraform Cloud should use in the workspace. If left blank, Terraform Cloud uses the repository's default branch.
+This setting designates which branch of the repository HCP Terraform should use when the workspace is set to [Always Trigger Runs](#always-trigger-runs) or [Only trigger runs when files in specified paths change](#only-trigger-runs-when-files-in-specified-paths-change). If you leave this setting blank, Terraform Cloud uses the repository's default branch. If the workspace is set to trigger runs when a [git tag is published](#trigger-runs-when-a-git-tag-is-published), all tags will trigger runs, regardless of the branch specified in this setting.
```
Comment (rkoron007) [inline website/docs/cloud-docs/workspaces/settings/vcs.mdx:90]:
```suggestion
This setting designates which branch of the repository HCP Terraform should use when the workspace is set to [Always Trigger Runs](#always-trigger-runs) or [Only trigger runs when files in specified paths change](#only-trigger-runs-when-files-in-specified-paths-change). If you leave this setting blank, HCP Terraform uses the repository's default branch. If the workspace is set to trigger runs when a [git tag is published](#trigger-runs-when-a-git-tag-is-published), all tags will trigger runs, regardless of the branch specified in this setting.
```

**Review by rkoron007 — APPROVED** [review]
Looks good to me! Merging this in

**Pull request merged**